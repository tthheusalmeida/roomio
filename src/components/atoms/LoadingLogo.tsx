import LoadingLabel from "./LoadingLabel";

export default function LoadingLogo() {
  return <LoadingLabel label="Room.io" className="font-bold text-6xl" />;
}
