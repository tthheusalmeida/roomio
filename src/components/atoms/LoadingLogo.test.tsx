import { render, screen } from "@testing-library/react";
import LoadingLogo from "./LoadingLogo";

describe("Loading component", () => {
  it("renders the LoadingLabel with correct text and classes", () => {
    const { asFragment } = render(<LoadingLogo />);

    const label = screen.getByText("Room.io");
    expect(label).toBeInTheDocument();
    expect(label).toHaveClass("animate-pulse", "font-bold", "text-6xl");

    expect(asFragment()).toMatchSnapshot();
  });
});
