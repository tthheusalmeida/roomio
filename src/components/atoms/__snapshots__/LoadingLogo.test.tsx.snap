// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`Loading component renders the LoadingLabel with correct text and classes 1`] = `
<DocumentFragment>
  <span
    class="animate-pulse font-bold text-6xl"
  >
    Room.io
  </span>
</DocumentFragment>
`;
